src/hooks/useAppHooks.js

```diff
@@ -1,13 +1,36 @@
 import { useEffect, useRef, useState } from 'react'
```

Comment:
Should we maybe rename this file to `useOrgHooks`? because we have a mix of differents apps and org hooks here, not sure but what do you think?

Reply:
Yeah that makes sense. And we will most likely be adding more data from different apps.